src/nl802154.h

```diff
@@ -112,6 +117,33 @@ enum nl802154_attrs {
 
 	NL802154_ATTR_SUPPORTED_COMMANDS,
 
+	NL802154_ATTR_BEACON_INDICATION_TIMEOUT,
```

Comment:
Please put additional attributes at the bottom of the list, where it says "/* add attributes here, update the policy in nl802154.c */"